content/desktop/release-notes.md

```diff
@@ -24,11 +24,60 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.22.0
+
+{{< release-date date="2023-08-03" >}}
+
+{{< desktop-install all=true version="4.22.0" build_path="/117440/" >}}
+
+### Upgrades
+- [Compose v2.21.0](https://github.com/docker/compose/releases/tag/v2.21.0)
+- [Docker Engine v24.0.6](https://docs.docker.com/engine/release-notes/24.0/#2406)
+- [Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1). With Docker Scout, you can now:
+  - You can now manage temporary and cached files with `docker scout cache`.
+  - You can now manage environments with `docker scout environment`.
+  - You can now configure default organization with `docker scout config`.
+  - List packages of an image with their vulnerabilities with `docker scout cves --format only-packages`.
+  - You can enroll an organization with Docker scout with `docker scout enroll`.
+  - You can now stop, analyse and compare local file system with `docker scout cves --type fs`.
+- Wasm runtimes:
+  - wasmtime, wasmedge revision `d0a1a1cd`.
+  - slight and spin wasm `v0.9.0`.
+
+### New
+
+- Added support for new Wasm runtimes: wws and lunatic.
```

Comment:
```suggestion
- Added support for new Wasm runtimes, wws and lunatic.
```

Reply:
here the Wasm runtimes are wws and lunatic, do we still want to change to the comma?